core/services/fluxmonitorv2/flux_monitor.go

```diff
@@ -655,25 +662,41 @@ func (fm *FluxMonitor) respondToNewRoundLog(log flux_aggregator_wrapper.FluxAggr
 	}
 
 	// Call the v2 pipeline to execute a new job run
-	runID, answer, err := fm.pipelineRun.Execute(metaDataForBridge)
+	run, results, err := fm.runner.ExecuteRun(context.Background(), fm.spec, pipeline.JSONSerializable{Val: metaDataForBridge}, *fm.logger)
```

Comment:
context.TODO ?

Reply:
See comment above - I think we do want these to complete